spec/schema-mastery_criteria.json

```diff
@@ -15,7 +15,44 @@
         "num_correct_in_a_row_2",
         "num_correct_in_a_row_3",
         "num_correct_in_a_row_5",
-        "num_correct_in_a_row_10"
+        "num_correct_in_a_row_10",
+        "pre_post_test"
+      ]
+    },
+    "pre_post_test": {
+      "type": "object",
+      "description": "Definition for pre/post test",
+      "additionalProperties": false,
```

Comment:
Setting `additionalProperties` to false might make it harder to extend or add extra metadata later on for the Pre/Post tests, because we'd have to update the schema each time. Not a major concern, but since this is a pilot feature, should we avoid setting this for now? @rtibbles